chapters/Chapter 2 — Dressara.md

```diff
@@ -0,0 +1,94 @@
+The women of Dressara often wore dresses, which Leanne both admired and found a shame. Leanne enjoyed wearing dresses, but found women in suits much more handsome.
+
+Delphine was nothing if not handsome. She was tall, and did her best to look so: her fine Elisian suit lengthened her torso, and she had paired it, rather than with pants, with a skirt just long enough to wave gently in the light breeze the Dressarans were so fond of.
```

Comment:
In general, I really like this: it highlights her choices more than her appearance. However, certain things don't sit right with me:

> She was tall, and did her best to look so:

The "did her best to look so" feels a bit awkward to me. No better wording comes to mind, however.

> her fine Elisian suit lengthened her torso,

I like this idea. I don't like the word torso. 

> and she had paired it, rather than with pants, with a skirt just long enough to wave gently in the light breeze the Dressarans were so fond of.

This part I like.